packages/sw-routing/src/lib/router.js

```diff
@@ -122,7 +71,7 @@ class Router {
 
       let responsePromise;
       let matchingRoute;
-      for (let route of (routes || [])) {
+      for (let route of this._routes) {
         if (route.method !== event.request.method) {
```

Comment:
Yeah, we don't have support for `all`. I can switch to keeping track of routes per-method to optimize the loop.